source/_views/issue.html

```diff
@@ -0,0 +1,16 @@
+{% extends "default" %}
+{% block content_wrapper %}
+<article>
+    <div class="article container">
+        <div class="col-md-9">
+            <header>
+                <h1 class="pio-docs-title">{{ page.title }}</h1>
+              </header>
+            <div style="margin-top:15px;">
+                {{ page.description }}
+                {{ page.blocks.content|raw }}
+            </div>
+        </div>
```

Comment:
Suggest adding the [sidebar](https://github.com/pantheon-systems/documentation/pull/991/files#diff-3333494977238a0817e86bfb7d214f2fR20) under this line, so issue queues are represented on every page in the subdir.

Bonus, it will make short issues/descriptions look normal
### Without
![screen shot 2015-09-29 at 10 42 35 am](https://cloud.githubusercontent.com/assets/10119525/10169507/febb02c8-6696-11e5-9b49-3fe6c5fa15a8.png)

### With
![screen shot 2015-09-29 at 10 42 12 am](https://cloud.githubusercontent.com/assets/10119525/10169511/048f409c-6697-11e5-856b-45dd9f8cd0bd.png)

Also suggest editing the sidebar CSS